opentracing-api-extensions-tracerconverter/src/main/java/io/opentracing/contrib/api/tracer/converter/APIExtensionsTracerConverter.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright 2017 The OpenTracing Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.opentracing.contrib.api.tracer.converter;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.ServiceLoader;
+
+import io.opentracing.Tracer;
+import io.opentracing.contrib.api.TracerObserver;
+import io.opentracing.contrib.api.tracer.APIExtensionsTracer;
+import io.opentracing.contrib.tracerresolver.TracerConverter;
+
+/**
+ * This implementation of the {@link TracerConverter} interface is used to create an
+ * {@link APIExtensionsTracer} wrapper, if one or more {@link TracerObserver} implementation can
+ * be resolved using service loadable {@link TracerObserverResolver}s.
+ *
+ */
+public class APIExtensionsTracerConverter implements TracerConverter {
```

Comment:
Maybe rename to `APIExtensionsTracerDecorator` and still implement `TracerConverter` (inspired by @pavolloffay's comment) ?